cypress/integration/people.js

```diff
@@ -15,11 +15,11 @@ describe('People', () => {
     it('All tabs work', () => {
         cy.get('.form-control').type('has:email').type('{enter}').should('have.value', 'has:email')
         cy.wait(200)
-        cy.get('.ant-tabs-nav-list > :nth-child(2)', { timeout: 7000 }).click()
+        cy.get('.ant-tabs-nav-list > :nth-child(2)').click()
```

Comment:
Any way you could convert these to use data-attr? 
```
<Someelement data-attr="our-custom-attr" .../>
```
This will keep the components more flexible and when you're refactoring React code you'll be able to see what data-attr's need to be reidentified.

Reply:
Yeah this makes perfect sense - I had left comments for myself to do it later (because there were no data-attr for that element yet), and when the tests passed I just forgot to go back and do it. Done now!

Reply:
awesome!